src/helm/benchmark/server.py

```diff
@@ -28,6 +33,27 @@ def serve_config():
         )
 
 
+# Shim for running helm-server for old suites from old version of helm-summarize
```

Comment:
Could you state the context more explicitly: we used to use schema.yaml (now schema_classic.yaml), but now we use schema.json...

Reply:
OK, updated the comment.